core/domain/exp_services.py

```diff
@@ -792,6 +793,15 @@ def _save_exploration(committer_id, exploration, commit_message, change_list):
                 'which is too old. Please reload the page and try again.'
                 % (exploration_model.version, exploration.version))
 
+    if feconf.ENABLE_STRING_CLASSIFIER:
```

Comment:
Any reason not to do this at the end, together with the other `if feconf.ENABLE_STRING_CLASSIFIER` block?